data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -154,6 +164,28 @@ public static ExportTraceServiceRequest getExportTraceServiceRequest(List<Resour
                 .build();
     }
 
+    private List<ResourceSpans> getResourceSpansBatch(final List<EndToEndTestSpan> dataList) {
```

Comment:
nit - Now that the class has been renamed, would suggest something like `testSpanList`. Also applies to the for loop below.